Kernel/Process.cpp

```diff
@@ -2219,6 +2219,50 @@ KResult Process::do_killpg(pid_t pgrp, int signal)
     return error;
 }
 
+KResult Process::do_killall(int signal)
+{
+    InterruptDisabler disabler;
+
+    pid_t init_pid = 1;
+    bool any_succeeded = false;
+    KResult error = KSuccess;
+
+    // Send the signal to all processes we have access to for.
+    for (auto& process : *g_processes) {
+        // POSIX specifies we skip init.
+        if (process.pid() == init_pid)
+            continue;
```

Comment:
We don't have an `init`. The closest thing we have is `SystemServer` but that's not guaranteed to have a specific PID. Also it's fine to send signals to `SystemServer` if you wanted to.

This line from POSIX makes it sound like this is an optional behavior regardless, so I think we should just skip it:
>The unspecified processes to which a signal cannot be sent may include the scheduler or init.

Reply:
done